Source/WebCore/platform/PlatformScreen.cpp

```diff
@@ -26,7 +26,7 @@
 #include "config.h"
 #include "PlatformScreen.h"
 
-#if PLATFORM(COCOA) || PLATFORM(GTK)
+#if PLATFORM(COCOA) || PLATFORM(GTK) || (PLATFORM(WPE) && ENABLE(WPE_PLATFORM))
```

Comment:
We could make the guard be only `ENABLE(WPE_PLATFORM)` because that implies that `ENABLE(WPE)` is also enabled:

```suggestion
#if PLATFORM(COCOA) || PLATFORM(GTK) || ENABLE(WPE_PLATFORM)
```

Reply:
Yes, I thought about this, but I found a bit weird to use an ENABLE macro specific to a platform without the explicit platform check. The WPE_PLATFORM macro will be removed eventually and only the platform check would be used.

Reply:
Fair point 👌🏼